exercises/practice/phone-number/.meta/config.json

```
@@ -1,6 +1,15 @@
 {
   "blurb": "Clean up user-entered phone numbers so that they can be sent SMS messages.",
-  "authors": [],
+  "authors": [
+    "devkabiir"
+  ],
+  "contributors": [
+    "ErikSchierboom",
+    "jackhughesweb",
+    "kytrinyx",
+    "rebelwarrior",
+    "Stargator"
+  ],
```

```suggestion
  "contributors": [
  ],
```

---

I would suggest the following contributors due to updates to the documentation, refactoring tickets, addition of error handling, and updates for NNBD

```suggestion
  "contributors": [
    "kytrinyx",
    "Stargator"
  ],
```